source/data/tier1data/clients/icommands.rst

```diff
@@ -0,0 +1,291 @@
+.. _icommands:
+
+#########
+iCommands
+#########
+
+
+iCommands is a command-line interface for iRODS, the open-source
+software behind Tier-1 Data. For those who are familiar with Unix command-line
+programs, it is a powerful and easy to use tool.
+
+************
+Installation
+************
+
+iCommands are already installed on the following HPC clusters:
+
+- Genius
+- wICE
+- Hydra
```

Comment:
I suggest to use the same format as in https://docs.vscentrum.be/jobs/running_jobs.html
```suggestion
.. grid:: 2
    :gutter: 4

    .. grid-item-card:: KU Leuven/UHasselt
       :columns: 12 4 4 4

       * Tier-2 :ref:`Genius <Genius hardware>`
       * Tier-2 :ref:`wICE <wICE hardware>`

    .. grid-item-card:: VUB
       :columns: 12 4 4 4

       * Tier-2 :ref:`Hydra <Hydra hardware>`
```

Reply:
Applied, pushing soon